tests/NHSD.BuyingCatalogue.Identity.Api.Testing.Data/Entities/DataProtectionKeys.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Testing.Data.Entities
+{
+    public static class DataProtectionKeys
+    {
+        public static async Task<IReadOnlyList<DataProtectionKey>> GetFromDb(string connectionString)
+        {
+            await using var databaseConnection = new SqlConnection(connectionString);
```

Comment:
Would it be possible to expand the `SqlRunner` class so that all our database connection logic is localised to one file? So my thinking here is to add a `QueryAsync` option unless this method can utilise the `FetchSingleResultAsync` method that already exists on the `SqlRunner` class.